py_feels/binary/binary_classifier.py

```diff
@@ -12,25 +12,31 @@
 
 # we only need pandas, mongo and joblib to use the model
 import pandas as pd
+import datetime
+from sqlalchemy import create_engine
 from sklearn.externals.joblib import load
 
 # load fitted model
 model = load('./model/binary_classifier.pckl')
 
-# read amazon training data for testing loaded model
-amazon = pd.read_csv(
-        './training/UCI_training_data/amazon_cells_labelled.txt', 
-        sep="\t", 
-        names=['sentence', 'sentiment'], 
-        dtype={'sentence':str, 'sentiment':int}
-)
+# connect to  SQL database, query for last item in 'tweets' table
+engine = create_engine('postgresql+psycopg2://wojak:@localhost/bit_feels_dev')
+query  = "SELECT * FROM tweets WHERE id=(SELECT max(id) FROM tweets)"
```

Comment:
Multiple `SELECT` should be avoided. Using `id` would work but we should probably use the timestamps. 

This should work:
`select text from tweets order by inserted_at desc limit 1;`